src/state/zustand/swapperStore/useSwapperStore.ts

```diff
@@ -67,6 +69,9 @@ export const useSwapperStore = (() => {
             set,
             'availableSwappersWithMetadata',
           ),
+          handleAssetToggle: handleAssetToggle(set, getState),
+          updateSelectedCurrencyToUsdRate: createUpdateAction(set, 'selectedCurrencyToUsdRate'),
+          handleInputAmountChange: handleInputAmountChange(set, getState),
```

Comment:
can we get a tl;dr on the rationale of dep injecting the `getState` accessor fn here?